app/views/devise/sessions/new.html.erb

```diff
@@ -95,6 +95,15 @@
           ),
         ) %>
   </p>
+  <% if FeatureManagement.sign_in_recaptcha_enabled? || true %>
```

Comment:
i know it's a draft but just wanted to triple check we don't leave this in before merging

Reply:
Can we include a spec that ensures that the text isn't rendered when the feature is disabled?

Reply:
I have added that spec.